### PR TITLE
Error when accessing achievement without being logged in

### PR DESCRIPTION
**Pull request by Maikdani**
Solves #3266 

Made the treema visible to be build, but disabled and hidden the buttons.

The: 
readOnly: me.get('anonymous')
In AchievementEditView.coffee

Seems to already make the Treema read only for admins if I'm correct? 
If you had something different in mind let me know.

### CODE REVIEW

**Comment by Maikdani** [comment]
adminonly
How it currently looks like

adminonly2

How it looks like after the PR.

File: app/templates/editor/achievement/edit.jade [hunk 1]
```diff
@@ -11,24 +10,27 @@ block content
...
     button.achievement-tool-button(data-i18n="common.save", disabled=(me.isAdmin() === true || me.isArtisan() === true) ? undefined : "true").btn.btn-primary#save-button Save
 
-    h3(data-i18n="achievement.edit_achievement_title") Edit Achievement
-      span
-        |: "#{view.achievement.attributes.name}"
+  else
+    button.achievement-tool-button(data-i18n="", disabled=true, style="display:none;").btn.btn-primary#recalculate-all-button Recalculate All
+    button.achievement-tool-button(data-i18n="", disabled=true, style="display:none;").btn.btn-primary#recalculate-button Recalculate
+    button.achievement-tool-button(data-i18n="common.delete", disabled=true, style="display:none;").btn.btn-primary#delete-button Delete
+    button.achievement-tool-button(data-i18n="common.save", disabled=true, style="display:none;").btn.btn-primary#save-button Save
```
Comment (sderickson) [inline app/templates/editor/achievement/edit.jade]: Why are these buttons in the template if they're both disabled and hidden?

File: app/templates/editor/achievement/edit.jade [hunk 2]
```diff
@@ -11,24 +10,27 @@ block content
...
 
-    #achievement-treema
+  h3(data-i18n="achievement.edit_achievement_title") Edit Achievement
+    span
+      |: "#{view.achievement.attributes.name}"
```
Comment (sderickson) [inline app/templates/editor/achievement/edit.jade]: Could you switch this to use `view.achievement.get('name')` instead? Not sure why the old version get it through the attributes property...

File: app/templates/editor/achievement/edit.jade (hunk 1 above)
Comment (Maikdani) [inline app/templates/editor/achievement/edit.jade]: Can indeed just remove them. Was told to hide/disable them so took that i bit too literally

**Comment by sderickson** [comment]
Just tested it, looks good! It's just missing some sort of feedback to users who can't edit achievements. Check out [ArticleEditView](https://github.com/codecombat/codecombat/blob/master/app/views/editor/article/ArticleEditView.coffee#L64), which shows a noty message. Add the same message to the AchievementEditView when the user is not authorized (not an admin or artisan) and then it should be good. Thanks!

**Comment by Maikdani** [comment]
That should do it, if I understood you correctly. :shipit:

**Comment by sderickson** [comment]
Looks good, thanks! :+1:

**Pull request merged**